Scripts/SL-Helpers.lua

```diff
@@ -484,6 +484,7 @@ SetGameModePreferences = function()
 	local prefix = {
 		ITG = "",
 		["FA+"] = "ECFA-",
+		["ECFA"] = "ECFA-",
```

Comment:
This should be something else, otherwise FA+ and ECFA scores will mix, and they have completely different windows and weights. ECFA2021 maybe?